### ISSUE TITLE
QviArt 4K not booting

### ISSUE THREAD

**Issue by petrkr** [OpenVisionE2/openvision-development-platform#103]
```
crashdate=Sun Jan 10 16:07:23 2021
compiledate=Jan 10 2021
skin=Default Skin
sourcedate=2021-01-10
branch=develop
rev=b31adea
component=enigma2

stbmodel=lunix4k
stbbrand=dags
stbplatform=dags72604
kernelcmdline=root=/dev/mmcblk0p4 rootwait rw rootwait=data=journal debug,nolock vmalloc=636m bmem=643m@380m hwtypenum=337
distro=openvision
oe=master
python=2.7.18
mediaservice=enigma2-plugin-systemplugins-servicemp3
multilib=False
architecture=cortexa15hf-neon-vfpv4
socfamily=bcm72604
visionversion=10.2
visionrevision=r326
visionlanguage=extralanguage
compiledby=petrkr
feedsurl=http://openvision.yacp.net/feeds/openvision


< 371.7559> 16:07:23.8730 [gFBDC] no framebuffer available
```

seems like driver / kernel problem maybe like before on sf4008 - #30  ?

**Comment by persianpros** [comment]
Yes another one ...

**Comment by persianpros** [comment]
[diff.zip](https://github.com/OpenVisionE2/openvision-development-platform/files/5800755/diff.zip)

If you have time please check the diff, maybe you could find what's changing the module layout again.

**Comment by petrkr** [comment]
Yup illl check, meanwhile i tried to got some info and this time there is no module layout field like in sf4008, it looks like it's something what must be compiled inside too. So now i do not know how to check kernel vs module compatibility without runtime.

For now I'll check diff and try to remember where I saw possible breaking option, but it would be like game 'try-mistake'.

Sad is in other working images is missing config.gz

Good is I found two testers. But they have 'combo' version of that box, but it seems it has same image, because there are no lunix4kco images like at lunix vs lunixco

**Comment by persianpros** [comment]
The image for lunix4k and lunix4kcombo is the same so they can use it too but first you need to find what's breaking the kernel :+1:
